tools/updateAndPatchBootstrap.R

```diff
@@ -1,38 +0,0 @@
-version <- "3.4.1"
```

Comment:
Would it make sense to keep this script, just without the patch stuff?

Reply:
I guess, but hopefully in the near future we'll be using `bootstraplib::bootstrap()` ;)

Reply:
(Also, even if we don't, I doubt it'll be that useful as BS3 is end-of-life)